staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter.go

```diff
@@ -117,10 +115,10 @@ func (c *converterImpl) FromUnstructured(u map[string]interface{}, obj interface
 		newObj := reflect.New(t.Elem()).Interface()
 		newErr := fromUnstructuredViaJSON(u, newObj)
 		if (err != nil) != (newErr != nil) {
-			glog.Fatalf("FromUnstructured unexpected error for %v: error: %v", u, err)
+			panic(fmt.Errorf("FromUnstructured unexpected error for %v: error: %v / %v", u, err, newErr))
```

Comment:
We use glog everywhere for logging everything. Panic prints to stderr IIRC, Fatalf prints to both stderr (as we have '--alsoprinttostderr` flag set) and a file, which is useful.